content/id/docs/contribute/participate/_index.md

```diff
@@ -52,7 +52,7 @@ Tim GitHub dan berkas OWNERS.
 Terdapat dua kategori tim dalam SIG Docs [tim (_teams_)](https://github.com/orgs/kubernetes/teams?query=sig-docs) dalam GitHub:
 
 - `@sig-docs-{language}-owners` merupakan pemberi persetujuan (_approver_) dan pemimpin (_lead_)
-- `@sig-docs-{language}-reviewers` merupakan pengulas (_reviewer_)
+- `@sig-docs-{language}-reviews` merupakan pengulas (_reviews_)
```

Comment:
```suggestion
- `@sig-docs-{language}-reviews` merupakan pengulas (_reviewer_)
```
@Arhell I don't think you need to change the later. Since no changes in the EN PR for that part (they keep it as `reviewers` too).

Reply:
Done